main.cpp

```diff
@@ -413,24 +485,29 @@ struct CargoSpace
     int capacity = 400;
     float loadingTime = 45;
 
+    CargoSpace();
+
     void transport();
     void store(int amountOfCargo);
     void secure(bool isDoorEnclosed);
 };
 
+CargoSpace::CargoSpace()
+{
+    std::cout << "CargoSpace being constructed!" << std::endl;
+}
+
 void CargoSpace::transport()
 {
     std::cout << "Cargo is being transported." << std::endl;
 }
 
 void CargoSpace::store(int amountOfCargo)
 {
-    if(amountOfCargo <= capacity)
-    {
+    if(amountOfCargo <= capacity) {
         std::cout << amountOfCargo << " units of cargo stored." << std::endl;
     }
-    else
-    {
+    else {
         std::cout << "Cargo exceeds maximum capacity." << std::endl;
     }
```

Comment:
Curly braces go on their own line with no other code
- [ ] revise: conform to coding standard